### ISSUE TITLE
Name Property incorrect for list items present under header GridView with Layout Customization

### ISSUE THREAD

**Issue by YuliKl** [microsoft/Xaml-Controls-Gallery#327]
**Describe the bug**

**Steps to reproduce the bug**

1. Launch XAML Controls Gallery.
2. Activate GridView button.
3. Navigate to list items present under header GridView with Layout Customization

**Expected behavior**

Correct name should be there.

**Actual behavior**

All list items have name like "AppUI basic"

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/72561901-0da22480-385f-11ea-9987-42429b4f765d.png)

**Version Info**

Gallery version (found on Settings page): 

1.2.12.0

**Additional context**

Copied from [internal bug](https://dev.azure.com/microsoft/OS/_workitems/edit/24650048)

**Comment by YuliKl** [comment]
@anawishnoff FYI
